command.o: In function `report_error':
command.c:(.text+0x5842): undefined reference to `WEXITSTATUS'

Index: src/command.c
--- src/command.c.orig
+++ src/command.c
@@ -115,6 +115,7 @@ int thread_rl_RetCode = -1; /* return code from readli
 
 #ifndef _Windows
 # include "help.h"
+# include <sys/wait.h>
 #endif /* _Windows */
 
 #ifdef _Windows
